sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -140,11 +158,11 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
 		.sof_tplg_filename = "sof-tgl-rt711-rt1308.tplg",
 	},
 	{
-		.link_mask = 0x1, /* this will only enable rt5682 for now */
+		.link_mask = 0x3, /* rt5682 on link0 and 2xmax98373 on link 1 */
 		.links = tgl_chromebook_base,
 		.drv_name = "sof_sdw",
 		.sof_fw_filename = "sof-tgl.ri",
-		.sof_tplg_filename = "sof-tgl-rt5682.tplg",
+		.sof_tplg_filename = "sof-tgl-rt5682-mx8373.tplg",
```

Comment:
You may actually keep the previous solution below, so that by just changing the link mask you can debug rt5682 on its own.

Reply:
Sure.. Shall add rt5682+max98373 as a seperate entry